### PR TITLE
remove padding from viewbox

### PR DESCRIPTION
**Pull request by kineticz**

### CODE REVIEW

File: src/svg_avatar_builder.rs [hunk 1]
```diff
@@ -166,7 +166,7 @@ impl SvgAvatarBuilder {
     }
 
     fn create_document() -> Document {
-        Document::new().set("viewBox", (-1.1, -1.1, 2.3, 2.3))
+        Document::new().set("viewBox", (-1, -1, 2, 2))
```
Comment (nathiss) [inline src/svg_avatar_builder.rs:169]: Hey @kindlychung,

thanks for the PR. I did initially create the `viewBox` without the padding, but then stroke lines get cut off:

image

I was trying to find a solution and it seems like allowing overflow to be visible on the `svg` tag fixes it.

```rust
fn create_document() -> Document {
    Document::new()
        .set("viewBox", (-1, -1, 2, 2))
        .set("overflow", "visible")
}
```

What do you think about it? If it coves your use case, then I'll be happy to merge the changes for you!

**Comment by nathiss** [comment]
Hey @kindlychung,

FYI, `svg_avatars` with your changes is available on crates.io: https://crates.io/crates/svg_avatars. Thanks for the contribution!

File: src/svg_avatar_builder.rs (hunk 1 above)
Comment (kineticz) [reply src/svg_avatar_builder.rs:169]: Complete fine with it, thanks for looking into it!

**Comment by kineticz** [comment]
@nathiss My pleasure!

**Pull request merged**